core/src/main/java/org/apache/struts2/result/PostbackResult.java

```diff
@@ -76,6 +76,10 @@
  */
 public class PostbackResult extends StrutsResultSupport {
 
+    /**
+     * 
+     */
+    private static final long serialVersionUID = -2283504349296877429L;
```

Comment:
Could you drop that empty comment section? And please add an empty line to separate static field from the other fields. Thanks :)